data/tokens.json

```
@@ -410,6 +410,40 @@
           "value": "900",
           "type": "fontWeights"
         }
+      },
+      "mobileSize": {
+        "h1": {
+          "value": "32px",
+          "type": "fontSizes"
+        },
+        "h2": {
+          "value": "28px",
+          "type": "fontSizes"
+        },
+        "h3": {
+          "value": "22px",
+          "type": "fontSizes"
+        },
+        "h4": {
+          "value": "20px",
+          "type": "fontSizes"
+        },
+        "h5": {
+          "value": "16px",
+          "type": "fontSizes"
+        },
+        "body": {
+          "value": "16px",
+          "type": "fontSizes"
+        },
+        "footnode": {
```

Sorry I commented on the wrong file! `transformed-tokens.json` is auto-generated 😅 

```suggestion
        "footnote": {
```

---

😂